_sass/footer_minimal.scss

```diff
@@ -0,0 +1,81 @@
+
+// variable for fonts
+$font_color:#686b6f;
+
+#social_icon {
+    color: #686b6f;
+}
+// property for <hr>  for footer
+#break_line{
+//  background-color: $font_color;
+    border: none;
+     width: 100%;
```

Comment:
Consistent spacing issue here and could remove the blank lines at top (minor nit picks :))